spec/cases/each_with_ar_sqlite.rb

```diff
@@ -29,11 +27,19 @@ class User < ActiveRecord::Base
 print "Parent: "
 puts User.first.name
 
+# https://www.sqlite.org/faq.html
+# Under Unix, you should not carry an open SQLite database across a fork() system call into the child process.
+ActiveRecord::Base.connection.disconnect!
+
 print "Parallel (#{in_worker_type}): "
 Parallel.each([1], in_worker_type => 1) do
+  ActiveRecord::Base.establish_connection(params)
```

Comment:
I tried both, and the  ActiveRecord::Base.connection_pool.with_connection do technique, but this was the only way that works with both 0 and non-zero in_worker_type => N